news/2018-09-09-project-loved-week-of-september-9th.md

```diff
@@ -0,0 +1,230 @@
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/dziekuje.jpg)](https://osu.ppy.sh/community/forums/topics/801923)
+
+[Doda - Dziekuje](https://osu.ppy.sh/beatmapsets/659007) by [-NeBu-](https://osu.ppy.sh/users/4099626)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+A wonderfully bright song paired with exciting 7\* jumps marks -NeBu-'s latest energetically Polish creation. It may be one of the most delightful jump maps I've had the pleasure of playing as of late, from the song choice to the jump patterns, to the pretty star sliders littered throughout the map. It's received a deserved 160+ favorites in around a year, and for a song about overcoming fear and flying to the stars, these feelings perfectly parallel the euphoria of actually hitting all the jumps.
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/grief-malice.jpg)](https://osu.ppy.sh/community/forums/topics/801924)
+
+[Helblinde - Grief & Malice](https://osu.ppy.sh/beatmapsets/256997) by [Necroluttah](https://osu.ppy.sh/users/2917471)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+If you've been a fan of the harsher jump maps that have been included in the Loved section, but just aren't quite there yet in skill to play them, Necroluttah's Grief & Malice from 2015 is sure to scratch that same itch for you. High velocity sliders, not-quite-so-fullscreen jump patterns, and a hot track from the wonderful Featured Artist [Helblinde](https://osu.ppy.sh/beatmaps/artists/5) all mesh together for a 160+ favorited cocktail of hype-ass mapping.
+
+[![](/wiki/shared/news/2018-09-09-project-loved-week-of-september-9th/osu/tool-assisted-speedcore-tqbf-frame-advance-rmx.jpg)](https://osu.ppy.sh/community/forums/topics/801925)
+
+[Kobaryo - Tool-Assisted Speedcore (TQBF Frame Advance RMX)](https://osu.ppy.sh/beatmapsets/782508) by [Mismagius](https://osu.ppy.sh/users/19048)  
+*written by [Toy](https://osu.ppy.sh/users/2757689)*
+
+**Can you please play Big Black?**. These words have haunted popular players for six long years, and now Mismagius has finally brought us a glimpse of what could be Big Black 2. It's rougher, meaner, and overall a hell of a lot less forgiving. The slider jumps are quicker, the streams are more spaced, the buzz sliders are faster so for those of you who wished The Big Black could have offered a bit more of a challenge, you'll find exactly what you're looking for here.
```

Comment:
this description switches between referring to The Big Black by "Big Black" and "The Big Black", suggest sticking with the later one to be consistent, and it would sound better grammatically for the initial quote